### DISCUSSION TITLE
Timeline for v4.5.0

### DISCUSSION THREAD

**Discussion by jayanthchennamaneni** [do-mpc/do-mpc#348] [General]
hello,
I have been following do-MPC for quite while now
Just wanted to know what is the timeline for v.4.5.0/ when it will be out, curious about implementing NN based MPC for a research project.

Thank you and keep up the good work of clean and simple code.
CJ

**Comment by 4flixt** [comment]
Hi @jayanthchennamaneni ,

thank you for your interest in v4.5.0 :) We are also very excited about the new features. We are still doing some testing and polishing with the new version but expect that it will launch until 10.03. 

Best,
4flixt

**Comment by jayanthchennamaneni** [comment]
Hi @4flixt,

Will be looking out for it.
Good luck for the next version.

Thanks,
CJ
